### PR TITLE
CXP-1363: Update the header menu

### PR DESCRIPTION
**Pull request by hogren**
Expected: 
- Update the app header menu to match the last version of the mockups
- when I click on help, I land on https://apps.akeneo.com/extension/akeneo-demo-app
- when I click on Go to Akeneo PIM, I land on the connected app page 
:warning: the link has changed. Please add /#/connect/connected-apps/[app_id] after the main PIM URL.

Preview : 
image

### CODE REVIEW

**Comment by tseho** [comment]
Hello Florian, in your screenshot the link "Help" has neither the correct color nor the icon.

**Comment by hogren** [comment]
> Hello Florian, in your screenshot the link "Help" has neither the correct color nor the icon.
it's modified.

**Review by RiffVentura — APPROVED** [review]

File: templates/connected-base.html.twig [hunk 1]
```diff
@@ -6,13 +6,10 @@
             <img class="header__logo" src="{{ asset('build/images/logo-demoapp.svg') }}"
                  alt="{{ 'page.welcome.connect.logo_alt' | trans }}" />
             <nav class="main-navigation">
-                <a href="{{ path('products') }}" class="main-navigation__link main-navigation__link--active" >
-                    {{ 'common.header.links.catalog' | trans }}
-                </a>
-                <a href="https://marketplace.akeneo.com/extension/akeneo-demo-app" target="_blank" class="main-navigation__link" >
+                <a href="https://marketplace.akeneo.com/extension/akeneo-demo-app" target="_blank" class="main-navigation__link main-navigation__link--external" >
```
Comment (masacc) [inline templates/connected-base.html.twig:9]:
```suggestion
                <a href="https://apps.akeneo.com/extension/akeneo-demo-app" target="_blank" class="main-navigation__link main-navigation__link--external" >
```

File: templates/connected-base.html.twig [hunk 2]
```diff
@@ -6,13 +6,10 @@
...
                     {{ 'common.header.links.help' | trans }}
                 </a>
-                <a href="{{ app.session.get('pim_url') }}" target="_blank" class="main-navigation__link main-navigation__link--external">
+                <a href="{{ app.session.get('pim_url') }}/connect/apps/v1/catalogs/{{ catalog.id }}" target="_blank" class="main-navigation__link main-navigation__link--external">
```
Comment (masacc) [inline templates/connected-base.html.twig:12]: The link is incorrect when you look a the ticket:
```suggestion
                <a href="{{ app.session.get('pim_url') }}/#/connect/connected-apps/{{ connected_app.id }}" target="_blank" class="main-navigation__link main-navigation__link--external">
```

I don't know if we store the connected app id somewhere, maybe it should be hard-coded as it is a fixed id coming from the app store (you may have to dig a little bit on that)

File: tests/Integration/Controller/ListProductsActionTest.php [hunk 3]
```diff
@@ -43,15 +43,16 @@ public function itDisplaysTenProductsAndTheCurrentLocale(): void
      */
     public function itRendersALinkThatTargetThePimUrl(): void
```
Comment (masacc) [inline tests/Integration/Controller/ListProductsActionTest.php:44]:
```suggestion
    public function itRendersALinkThatTargetThePimConnectedAppUrl(): void
```

File: templates/connected-base.html.twig (hunk 1 above)
Comment (masacc) [reply templates/connected-base.html.twig]: Just to explain: marketplace was the old name of the "Akeneo app store" so this hostname is deprecated.

**Review by masacc — APPROVED** [review]

**Review by eoliveauakeneo — APPROVED** [review]

**Pull request merged**